js/weather-api/select.js

```diff
@@ -0,0 +1,21 @@
+let select = document.getElementById('selectCountry');
```

Comment:
constで宣言できる変数はconstで宣言しましょう。
```suggestion
const select = document.getElementById('selectCountry');
```